tests/uvdata/test_mir_parser.py

```diff
@@ -766,7 +768,7 @@ def test_apply_tsys_missing_recs(mir_data):
         mir_copy.load_data(load_cross=True, apply_tsys=True)
 
     for key in mir_data.vis_data:
-        assert np.allclose(
+        np.testing.assert_allclose(
             mir_data.vis_data[key]["data"], mir_copy.vis_data[key]["data"]
         )
```

Comment:
Do we have any tols acceptable for this check? All the other tests have explicit tols. or are these values integers? i suppose in that case we won't need it.

Reply:
This is on a mir_data object so I defer to @kartographer

Reply:
This one _should_ be able to be treated as a straight-up equality -- it's functionally a no-op under the hood because no calibration information (namely, system temperature information) can be found, hence the check below that all vis records are flagged.

Reply:
So I get the sense it doesn't really matter this line then.

Reply:
Not _this_ time :wink: